collector/diskstats_linux.go

```diff
@@ -185,6 +185,11 @@ func (c *diskstatsCollector) Update(ch chan<- prometheus.Metric) error {
 			if err != nil {
 				return fmt.Errorf("invalid value %s in diskstats: %s", value, err)
 			}
+			// Convert to seconds
+			switch i {
+			case 3, 7, 9, 10:
```

Comment:
That's pretty prone to errors. Would be great to use a value function instead, similar to what is done already in diskstats_darwin.go.

Reply:
Done

Reply:
Thanks!